src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -1648,6 +1676,18 @@ static int parse_args(vector<const char*>& args, std::ostream *err_msg,
         return -EINVAL;
       }
       cfg->set_max_part = true;
+    } else if (ceph_argparse_flag(args, i, "--persist", (char *)NULL)) {
+      cfg->persist = true;
+    } else if (ceph_argparse_witharg(args, i, &cfg->persist_timeout, err,
```

Comment:
For the `persist_timeout` here you may should set up one default value if absent. 30 seocnds is recommend.

Reply:
It is set to 60 by default (persist_timeout is initialized with this value).

Reply:
The default IO timer in kernel for nbd is 30 seconds. So the value >= 30 should be okay.